Sourcery/Generating/Template/JavaScript/JavaScriptTemplate.swift

```diff
@@ -38,7 +38,13 @@ final class JavaScriptTemplate: Template {
 
         let include: @convention(block) (String) -> String? = { [unowned self] path in
             let path = self.sourcePath.parent() + Path(path)
-            let includedTemplate: String? = try? path.read()
+            var includedTemplate: String? = try? path.read()
```

Comment:
Minor thing - I think we can just check right away if the path has extension and append it on demand, then try to read once.